arm-storage/2016-01-01/swagger/storage.json

```diff
@@ -562,7 +562,7 @@
         },
         "accessTier": {
           "type": "string",
-          "description": "Required for StandardBlob accounts. The access tier used for billing. Access tier cannot be changed more than once every 7 days (168 hours). Access tier cannot be set for StandardLRS, StandardGRS, StandardRAGRS, or PremiumLRS account types.",
+          "description": "Required for StandardBlob accounts. The access tier used for billing. Access tier can only be set for Blob Storage Accounts.",
```

Comment:
> "Required for StandardBlob accounts. The access tier used for billing. Access tier can only be set for Blob Storage Accounts."
- Which property are you referring to when you talk about "StandardBlob accounts"?
- Which property are you referring to when you talk about "Blob Storage Accounts"?

This is so confusing..